woocommerce/woocommerce-ios
# Product Creation AI: Remove old code for product from image

Closes: #10690 


## Description

This PR adds deprecated code related to adding product from image:
- Entry point from `AddProductCoordinator`
- A/B test setup
- Eligibility checker
- Support tag `ai_product_details_from_image`.
- Unit tests.

## Testing instructions

- Turn off the feature flag `productCreationAI` and build the app.
- Log in to a WPCom store and select adding product from store onboarding or the Products tab.
- Select add manually if the action sheet for template products is displayed.
- Select simple physical product type, notice the form for adding a simple physical product is displayed instead of the AI sheet.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/e7f13f3b-13e0-4ec3-bfb0-23498eb201cf



---
